phongo_compat.h

```diff
@@ -43,28 +43,6 @@
 # include <Zend/zend_string.h>
 #endif
 
-#if PHP_VERSION_ID >= 50500
-#define ITERATOR_GET_CURRENT_KEY(it, z) \
-		(it).funcs->get_current_key(&(it), z TSRMLS_CC);
-#else
-#define ITERATOR_GET_CURRENT_KEY(it, z) \
-{ \
-	char *str_key; \
-	uint str_key_len; \
-	ulong int_key; \
-	switch ((it).funcs->get_current_key(&(it), &str_key, &str_key_len, &int_key TSRMLS_CC)) { \
-		case HASH_KEY_IS_LONG: \
-			ZVAL_LONG(z, int_key); \
-			break; \
-		case HASH_KEY_IS_STRING: \
-			ZVAL_STRINGL(z, str_key, str_key_len-1, 0); \
-			break; \
-		default: \
-			RETURN_NULL(); \
-	} \
-}
-#endif
-
```

Comment:
It was introduced in cdc6471b46838d11bcf951a860ab22a893ef6f65 for `Cursor::key()` ([PHPC-130](https://jira.mongodb.org/browse/PHPC-130)) but later removed in 34fff68d850954f59c1c736b6b7bf6fe62355f53 ([PHPC-224](https://jira.mongodb.org/browse/PHPC-224)). This was all before 1.0.0. Best I can tell, it has been unused since.